theano/tensor/signal/downsample.py

```diff
@@ -235,14 +235,15 @@ def grad(self, inp, grads):
         x, = inp
         gz, = grads
         maxout = self(x)
-        if self.st != self.ds:
-            return [theano.gradient.grad_not_implemented(self, 0, x)]
         return [DownsampleFactorMaxGrad(self.ds,
                                         ignore_border=self.ignore_border,
                                         st=self.st)(
                                             x, maxout, gz)]
 
     def c_code(self, node, name, inp, out, sub):
+        "No implementation is currently for the case where
```

Comment:
This should be a comment (starting with `# `), not a docstring.
Also, currently it is a parse error, as multi-line strings should be enclosed in triple marks (`'''` or `"""`).